Homework1.Tests/MatrixTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace Homework1.Tests;
+
+public class Tests
+{
+    Matrix matrix1;
+    Matrix matrix2;
```

Comment:
```suggestion
    private Matrix matrix1;
    private Matrix matrix2;
```
В .NET по традиции всегда явно пишут модификаторы видимости, даже если умолчания устраивают